brian2/devices/cpp_standalone/templates/spatialneuron_prepare.cpp

```diff
@@ -1,7 +1,5 @@
-{# USES_VARIABLES { N, _invr, Ri, Cm, dt, area, r_length_1, r_length_2
+{# USES_VARIABLES { N, _invr, Ri, Cm, dt, area, r_length_1, r_length_2,
```

Comment:
Unrelated bug fix?

Reply:
It's actually only a cosmetic fix, our parsing does not require a comma at the end of the line.